dcpquery/db/__main__.py

```diff
@@ -45,6 +45,7 @@
     else:
         extractor_args = {"query": args.test_query}
 
-    DSSExtractor(staging_directory=".").extract(transformer=transform_bundle, loader=load_bundle, **extractor_args)
+    DSSExtractor(staging_directory=".").extract(
+        transformer=transform_bundle, loader=BundleLoad().load_bundle, **extractor_args)
```

Comment:
Consider this indentation style instead:
```
foo(bar=1,
    baz=2,
    **args)
```

Reply:
👍

Reply:
👍